activerecord/test/cases/multiparameter_attributes_test.rb

```diff
@@ -250,8 +250,8 @@ def test_multiparameter_attributes_on_time_only_column_with_time_zone_aware_attr
         }
         topic = Topic.find(1)
         topic.attributes = attributes
-        assert_equal Time.utc(2000, 1, 1, 16, 24, 0), topic.bonus_time
-        assert topic.bonus_time.utc?
+        assert_equal Time.zone.local(2000, 1, 1, 16, 24, 0), topic.bonus_time
+        assert_not topic.bonus_time.utc?
```

Comment:
I might be misunderstanding things, but won't this assertion fail if someone runs this test on a UTC machine?